roster_machine.js

```diff
@@ -9,17 +9,20 @@ const client = new Client({
 		Intents.FLAGS.GUILD_MEMBERS,
```

Comment:
I'm not sure what change it is that is causing it, but this updated bot code is crashing every 3-5 minutes.

![image](https://user-images.githubusercontent.com/14143235/160165108-23ca71c7-da41-49b5-8022-a3603c3f89d6.png)

The current code on the dev branch does not seem to share this issue.